manifests/c/Coq/CoqPlatform/Beta/8.14.0/Coq.CoqPlatform.Beta.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Coq.CoqPlatform.Beta
+PackageVersion: 8.14.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://github.com/coq/platform/releases/download/2021.09.0/Coq_Platform_2021.09.0.8.14+beta2-installer-windows-x86_64-signed.exe
+  InstallerSha256: 7C988D3BA76D0C3EC840A17974C6CD785439794B787F61D5673CDAC43FE4C2D8
+  ProductCode: Coq
```

Comment:
```suggestion
```
This will probably conflict with the stable version since the Product Codes are the same.